.eslintrc.js

```diff
@@ -82,4 +88,5 @@ module.exports = {
 			'@wordpress/data',
 		],
 	},
+	ignorePatterns: [ 'phpunit-html/**' ],
```

Comment:
I tripped over a local folder left over from testing and realised it should always be ignored for linting.